src/components/BoxSelection/docs/BoxSelectionWiringExample.tsx

```diff
@@ -0,0 +1,27 @@
+import * as React from 'react';
+import { BoxSelection } from '../';
+import { classes } from './BoxSelectionWiringExample.st.css';
+
+export const BoxSelectionWiringExample = () => {
+  const [selectedId, setSelectedId] = React.useState('1');
+  return (
+    <BoxSelection
+      name="hours"
+      className={classes.root}
+      onChange={({ id }) => {
+        return setSelectedId(id);
+      }}
+    >
+      {[1, 2, 3].map((_n, i) => (
+        <BoxSelection.Option
+          key={i}
```

Comment:
I didn't remove it since React yells at me that I must have a unique key prop on the map functionality.
if you still want me to remove it please write again.